pkg/openstack-manila/secret/secretsync.go

```diff
@@ -137,54 +138,14 @@ func (c *SecretSyncController) translateSecret(cloudSecret *v1.Secret) (*v1.Secr
 func cloudToConf(cloud clientconfig.Cloud) map[string][]byte {
 	data := make(map[string][]byte)
 
-	if cloud.AuthInfo.AuthURL != "" {
-		data["os-authURL"] = []byte(cloud.AuthInfo.AuthURL)
+	if cloud.Cloud != "" {
+		data["os-cloud"] = []byte(cloud.Cloud)
 	}
-	if cloud.RegionName != "" {
-		data["os-region"] = []byte(cloud.RegionName)
-	}
-	if cloud.AuthInfo.UserID != "" {
-		data["os-userID"] = []byte(cloud.AuthInfo.UserID)
-	} else if cloud.AuthInfo.Username != "" {
-		data["os-userName"] = []byte(cloud.AuthInfo.Username)
-	}
-	if cloud.AuthInfo.Password != "" {
-		data["os-password"] = []byte(cloud.AuthInfo.Password)
-	}
-	if cloud.AuthInfo.ApplicationCredentialID != "" {
-		data["os-applicationCredentialID"] = []byte(cloud.AuthInfo.ApplicationCredentialID)
-	}
-	if cloud.AuthInfo.ApplicationCredentialName != "" {
-		data["os-applicationCredentialName"] = []byte(cloud.AuthInfo.ApplicationCredentialName)
-	}
-	if cloud.AuthInfo.ApplicationCredentialSecret != "" {
-		data["os-applicationCredentialSecret"] = []byte(cloud.AuthInfo.ApplicationCredentialSecret)
-	}
-	if cloud.AuthInfo.ProjectID != "" {
-		data["os-projectID"] = []byte(cloud.AuthInfo.ProjectID)
-	} else if cloud.AuthInfo.ProjectName != "" {
-		data["os-projectName"] = []byte(cloud.AuthInfo.ProjectName)
-	}
-	if cloud.AuthInfo.DomainID != "" {
-		data["os-domainID"] = []byte(cloud.AuthInfo.DomainID)
-	} else if cloud.AuthInfo.DomainName != "" {
-		data["os-domainName"] = []byte(cloud.AuthInfo.DomainName)
-	}
-	if cloud.AuthInfo.ProjectDomainID != "" {
-		data["os-projectDomainID"] = []byte(cloud.AuthInfo.ProjectDomainID)
-	} else if cloud.AuthInfo.ProjectDomainName != "" {
-		data["os-projectDomainName"] = []byte(cloud.AuthInfo.ProjectDomainName)
-	}
-	if cloud.AuthInfo.UserDomainID != "" {
-		data["os-userDomainID"] = []byte(cloud.AuthInfo.UserDomainID)
-		data["os-domainID"] = []byte(cloud.AuthInfo.UserDomainID)
-	} else if cloud.AuthInfo.UserDomainName != "" {
-		data["os-userDomainName"] = []byte(cloud.AuthInfo.UserDomainName)
-		data["os-domainName"] = []byte(cloud.AuthInfo.UserDomainName)
-	}
-	if cloud.CACertFile != "" {
-		// Replace the original cert authority path from clouds.yaml with the canonical one
-		data["os-certAuthorityPath"] = []byte(cacertPath)
+
+	data["os-use-clouds"] = []byte(strconv.FormatBool(cloud.UseClouds))
+
+	if cloud.CloudsFile != "" {
+		data["os-clouds-file"] = []byte(cloud.CloudsFile)
```

Comment:
This indicates that path that the `clouds.yaml` can be found in. We're using this value in the Manila CSI driver, thus the value of this should be static and correspond to the path you are mounting the `clouds.yaml` file to.

Looking at your manifests from [an earlier commit](https://github.com/openshift/csi-operator/pull/373/commits/94000a8679fc362167ae1ee62da618a1c913d9c3#diff-f48ca1ef12471e0a934482bc5478ba4174bedef8677429c13815be4abed8ef7fR63), that file is currently `/etc/kubernetes/secret/clouds.yaml` (though I'm suggesting to change it). You should use the same static path here.